contractsV5/IKyberDAO.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity 0.5.11;
+
+interface IKyberDAO {
+    function getLatestNetworkFeeData() external view returns(uint feeInBps, uint expiryBlockNumber);
+    function getLatestBRRData() external view returns(uint burnInBps, uint rewardInBps, uint rebateInBps, uint epoch, uint expiryBlockNumber);
+    function claimStakerReward(uint epoch) external returns(uint);
+    function claimReserveRebate(uint epoch) external returns(uint);
+    function burnKNC() external returns(uint);
```

Comment:
I don't think DAO needs to know about burn KNC
it will be public in fee handler for anyone to call.

Reply:
okay, removing